distributed/asyncio.py

```diff
@@ -0,0 +1,61 @@
+import asyncio
```

Comment:
I'm not quite understand. The import in __init__.py whould be avoided and just `from distributed.asyncio import AioClient` ? Or put the import inside the functions?